### PR TITLE
Bumpup E2E test plugins matirx for v1.13

### PR DESCRIPTION
**Pull request by danfengliu**
Thank you for contributing to Velero!

# Please add a summary of your change
Add plugins default value for v1.13 E2E test.

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: test/util/velero/velero_utils.go [hunk 1]
```diff
@@ -96,6 +96,13 @@ var pluginsMatrix = map[string]map[string][]string{
 		"gcp":     {"velero/velero-plugin-for-gcp:v1.8.0"},
 		"csi":     {"velero/velero-plugin-for-csi:v0.6.0"},
 	},
+	"v1.13": {
+		"aws":     {"velero/velero-plugin-for-aws:v1.9.0"},
+		"azure":   {"velero/velero-plugin-for-microsoft-azure:v1.9.0"},
+		"vsphere": {"vsphereveleroplugin/velero-plugin-for-vsphere:v1.5.1"},
```
Comment (kaovilai) [inline test/util/velero/velero_utils.go:102]: Is vsphere plugin the only one that doesn't bump with new velero version?

Comment (blackpiglet) [reply test/util/velero/velero_utils.go:102]: There is a new tag https://github.com/vmware-tanzu/velero-plugin-for-vsphere/releases/tag/v1.5.2.
Not sure whether we should bump.

Comment (danfengliu) [reply test/util/velero/velero_utils.go:102]: Let me check with @yanji09.

Comment (danfengliu) [reply test/util/velero/velero_utils.go:102]: Got the confirmation, we can bump vSphere plugin to 1.5.2 in Velero v1.13, I will modify this PR once several successful nightly tests. Thanks @kaovilai.

**Review by ywk253100 — APPROVED** [review]

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**